var/spack/repos/builtin/packages/libevent/package.py

```diff
@@ -45,6 +45,8 @@ class Libevent(AutotoolsPackage):
     depends_on("automake", type="build")
     depends_on("libtool", type="build")
 
+    patch("autogen-no-which.patch")
```

Comment:
Does the patch apply to all versions? Also, did you consider just adding:
```python
depends_on("which", type="build")
```
?

Reply:
I did not consider that solution, that was very silly of me. Thank you for the excellent review, I'll take your suggestion instead.

Reply:
Hey, no worries! Thanks for the PR!